test/index.js

```diff
@@ -0,0 +1,8 @@
+const assert = require('assert');
```

Comment:
Please rename the `test` folder to `__tests__`

Reply:
Okay, It has been renamed, Please confirm

Reply:
Good.